src/ZendDeveloperTools/Module.php

```diff
@@ -213,4 +217,19 @@ public function getServiceConfig()
             ),
         );
     }
+    
+    /**
+     * 
+     * @return array
+     */
+    public function getAutoloaderConfig()
```

Comment:
actually, added it to make it look closer standard module. can be removed if required.